site/docs/v1/tech/release-notes.adoc

```diff
@@ -49,21 +49,37 @@ include::docs/v1/tech/__database-migration-warning.adoc[]
 
 === New
 * JWT Vending machine
-* IP Access Control. Requires Threat Detection feature.
+** This allows a JWT to be created for a not-yet-existing user with a payload defined by the API caller.
+** Resolves https://github.com/FusionAuth/fusionauth-issues/issues/525[GitHub Issue #525]
+* IP Access Control for API access or API keys
+** This allows domains or API addresses to be safelisted to use an API key.
+** Resolves https://github.com/FusionAuth/fusionauth-issues/issues/933[GitHub Issue #933]
+* FusionAuth wasn't awesome enough, so we added a robust Threat Detection feature for enterprise customers. This feature includes: 
+*** Rate Limiting
+*** Captcha
+*** IP Access Control Lists/blocking
+*** Blocked domains
 * Rate limiting per user for the following requests:
 ** Failed login (only used if Failed Login configuration is not in use)
 ** Forgot password
 ** Send email verification
 ** Send passwordless
 ** Send registration verification
 ** Send two-factor
-* Captcha. Requires Threat Detection feature.
-** Support for Google ReCaptcha v2, Google ReCaptcha v3, HCaptcha and HCaptcha Enterprise
-* IP location. Requires Threat Detection feature.
-* Impossible travel calculation during login. Requires Threat Detection feature.
-* More emails
+* Captcha - uses additional Threat Detection feature.
+** This adds (re)captcha for login pages using an email/password combination. 
+** Resolves https://github.com/FusionAuth/fusionauth-issues/issues/278[GitHub Issue #278]
+** Add support for Google ReCaptcha v2, Google ReCaptcha v3, HCaptcha and HCaptcha Enterprise
+* IP location 
+** Includes IP address in Forgot/Reset Password emails so the recipient can know where the request originated in case they did not request it. 
+** Flags suspicious IP addresses and sends an email to notify user of a new login with the IP's approximate location
+** With unexpected IP addresses, such as with geography, sends an email to notify the user of a new login with the approximate location of the IP address
+** Resolves https://github.com/FusionAuth/fusionauth-issues/issues/905[GitHub Issue #905]
+* Alert and track unusual user behavior around password resets
+** Resolves https://github.com/FusionAuth/fusionauth-issues/issues/1308[GitHub Issue #1308]
 * More webhook events
 ** Kickstart Success `kickstart.success`
```

Comment:
I just started this list, we should add all new events here and then link to the event doc